src/main/java/uk/gov/hmcts/cmc/claimstore/events/operations/ClaimantOperationService.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.cmc.claimstore.events.operations;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.config.properties.notifications.NotificationsProperties;
+import uk.gov.hmcts.cmc.claimstore.services.notifications.ClaimIssuedNotificationService;
+import uk.gov.hmcts.cmc.claimstore.stereotypes.LogExecutionTime;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+@Component
+@ConditionalOnProperty(prefix = "feature_toggles", name = "async_event_operations_enabled")
+public class ClaimantOperationService {
+    private final ClaimIssuedNotificationService claimIssuedNotificationService;
+    private final NotificationsProperties notificationsProperties;
+
+    @Autowired
+    public ClaimantOperationService(
+        ClaimIssuedNotificationService claimIssuedNotificationService,
+        NotificationsProperties notificationsProperties
+    ) {
+        this.claimIssuedNotificationService = claimIssuedNotificationService;
+        this.notificationsProperties = notificationsProperties;
+    }
+
+    public Claim notifyCitizen(Claim claim, String submitterName, String authorisation) {
+        //TODO check claim if operation already complete, if yes return claim else
+
+        claimIssuedNotificationService.sendMail(
+            claim,
+            claim.getSubmitterEmail(),
+            null,
```

Comment:
Can we please avoid passing `null` here? we ca introduce an overladed method with one less parameter in `ClaimIssuedNotificationService`

Reply:
Agreed to have a separate PR for this one.